jest.config.js

```diff
@@ -5,4 +5,7 @@ module.exports = {
   setupFilesAfterEnv: ['<rootDir>/.jest/jestDomSetup.ts'],
   testEnvironment: 'jsdom',
   globalSetup: '<rootDir>/.jest/globalSetup.ts',
+  transform: {
+    '\\.(js|jsx|tsx|ts)$': ['babel-jest', { configFile: './jest.babelrc' }],
```

Comment:
renamed `babelrc`, so that we only use this for `jest`. I was able to make `nextjs` to work with all defaults (no custom babel or webpack